doc.md

```diff
@@ -0,0 +1,110 @@
+# Hackseq 2017
+
+The system allows users to run graphical user interface application (e.g. `igv`), provides 100 GB hard memory and 20 GB soft memory, maps host users to Docker users inside containers and mounts team folders. 
+
+## Overview
+
+**Servers**: orca01.dmz.bcgsc.ca, orca02.dmz.bcgsc.ca
+* 5 teams per a server
+* a team shares a team’s folder
+* the Docker container per a team
+* each user has a home directory
+* ability to run GUI applications (X11)
+* the groups’ user name is the same as a name of the team
+
+## Group creation
+
+Add groups: `orca_users:*:1000:`, name of teams (e.g. `hackseq17_1:*:1002:`)
+
+## User creation
+* Primary group name = a name of the team = a name of the Docker container
+* Secondary group name = `1000(orca_users)`
+* User home directory: 
+    1) create empty `.Xauthority file` (X11)  
+    2) to make `brew install` work:
+    ```
+    mkdir -p -m775 ~/.cache/Homebrew 
+    ```
+    and copy  `.bashrc` from `/orca_repo/etc`
+* Make sure that in `.bashrc` there is the following line:
+```
+alias brew='sudo -u linuxbrew HOMEBREW_NO_AUTO_UPDATE=1 /home/linuxbrew/.linuxbrew/bin/brew'
+```
+* Add new entries from outputs of `getent passwd` and `getent group` commands to `group_hs17` and `passwd_hs17` files in `/orca_repo/etc`.
```

Comment:
Please provide a shell command for this.

Reply:
That's better than nothing, but I'd prefer you write a `sed` or `awk` script to do this task. It ought to be automated.

Reply:
I was planning to do that, but Brendan said that he is not going to integrate this script to the "user creation" script and he said that he will do that manually.

Reply:
Okay. Your description above is fine then.